Misc/NEWS.d/next/Library/2019-05-30-15-51-42.bpo-28724.34TrS8.rst

```diff
@@ -0,0 +1,2 @@
+The socket module now has the :func:`socket.send_fds()` and :func:`socket.recv.fds()` functions.

```

Comment:
It's a method, not a function, no? You should omit parenthesis.

The :class:`socket.socket` type gets 2 new methods: :meth:`socket.socket.send_fds` and :meth:`socket.socket.recv.fds`.